### DISCUSSION TITLE
Airflow Scheduler  Sqlalchemy Error: sqlalchemy.exc.PendingRollbackError: Can't reconnect until invalid transaction is rolled back.

### DISCUSSION THREAD

**Discussion by shivam708** [apache/airflow#39014] [Q&A]
My airflow scheduler gets killed due to an error  in run_loop_scheduler
sqlalchemy.exc.PendingRollbackError: Can't reconnect until invalid transaction is rolled back. 

I have tried changing the metadata db. Airflow db reset and db clean.
Got the same error in version 2.8.1 as well as 2.9.1
Sqlalchemy 1.4.59
Python 3.11
Meta Data DB: Postgres 12

**Comment by charankumar-procore** [comment]
is this resolved?. Getting same issue with 2.8.1
